papaparse.js

```diff
@@ -394,18 +394,29 @@
 			for (var row = 0; row < data.length; row++)
 			{
 				var maxCol = hasHeader ? fields.length : data[row].length;
-				var r = hasHeader ? fields : data[row];
 
-				if (skipEmptyLines !== 'greedy' || r.join('').trim() !== '')
+				var emptyLine = false;
+				var nullLine = hasHeader ? Object.keys(data[row]).length === 0 : data[row].length === 0;
+				if (skipEmptyLines && !hasHeader)
+					emptyLine = skipEmptyLines === 'greedy' ? data[row].join('').trim() === '' : data[row].length === 1 && data[row][0].length === 0;
```

Comment:
I know this library has a lot of un-braced conditionals. But I'd love to avoid that going forwards. Can you make sure all condition blocks in new code have `{}`?

Reply:
Gotcha.

Reply:
If you want to ensure this behaviour maybe we can include a rule in .eslintrc.js and fix all current dead code ;)